Widgets/WidgetsExtension.entitlements

```diff
@@ -2,6 +2,10 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+	<key>com.apple.developer.networking.networkextension</key>
+	<array>
+		<string>packet-tunnel-provider</string>
+	</array>
```

Comment:
Targets need to have the VPN entitlement in order to read/write changes. I'll be honest: I have no idea how this was working before this. But, myself and Jackson was able to reproduce occasional `permission denied` errors, and fixing the entitlements looks to have solved it completely. 🤷